doc/FAQ

```diff
@@ -0,0 +1,101 @@
+- NodeOS - Lightweight operating system using Node.js as userspace - FAQ -
+--------------------------------------------------------------------------
+
+
+1. NodeOS is a fantastic project, what can I do to help it?
+
+- Linux kernel programmers
+- Node.js programmers
+- Designers
+- Testers
+
+
+2. How to run NodeOS on my computer?
+
+- Running it on Qemu
+-- Download a prebuild version at https://github.com/NodeOS/NodeOS/releases
+-- Uncompress and execute run.sh
+-- Ports maped on your system to Qemu (50080 => 80, 50443 => 443)
+
+- Running it on Docker
+-- http://node-os.com/blog/get-involved
+-- Get NodeOS to access the internet
+--- ...
```

Comment:
I guess I sould here as well. The Qemu version do not install any packages, giving bunchs of errors, and some commands just give no output. The docker version give outputs to every shell command but after about two hous trying I quit from make it access the damn internet.